package.json

```diff
@@ -8,17 +8,14 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.1.2",
-    "@storybook/addon-centered": "^5.0.6",
-    "@storybook/addon-options": "5.0.6",
-    "@storybook/addon-viewport": "5.0.6",
-    "@storybook/react": "5.0.6",
+    "@storybook/addon-centered": "^5.3.2",
+    "@storybook/addon-options": "5.3.2",
+    "@storybook/addon-viewport": "5.3.2",
+    "@storybook/react": "5.3.2",
     "@types/lodash": "^4.14.118",
     "@types/node": "10.12.0",
     "@types/react": "^16.8.8",
     "@types/react-dom": "^16.8.4",
-    "@types/storybook__addon-centered": "^3.3.2",
```

Comment:
these packages were `This is a stub types definition. storybook__addon-centered provides its own type definitions, so you do not need this installed.` https://www.npmjs.com/package/@types/storybook__addon-centered